docker/intel/intel.cfg

```diff
@@ -4,7 +4,6 @@ CONTINUE_WITH_OPTIONAL_ERROR=yes
 PSET_INSTALL_DIR=/opt/intel
 CONTINUE_WITH_INSTALLDIR_OVERWRITE=yes
 PSET_MODE=install
-PHONEHOME_SEND_USAGE_DATA=no
 SIGNING_ENABLED=yes
-COMPONENTS=intel-icc-l-ps-ss-bec-wrapper__x86_64;intel-icc-l-all__x86_64;intel-compxe__noarch;intel-ccompxe__noarch;intel-psxe-licensing__noarch;intel-psxe-common__noarch
+COMPONENTS=DEFAULTS
```

Comment:
don't do that, it produces a huge images with stuff we don't need (GUI applications, Fortran compiler, etc.)